instrumentation/action_pack/Gemfile

```diff
@@ -8,15 +8,8 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'opentelemetry-api', path: '../../api'
-
 group :test do
-  gem 'byebug'
-  gem 'opentelemetry-common', path: '../../common'
   gem 'opentelemetry-instrumentation-base', path: '../../instrumentation/base'
   gem 'opentelemetry-instrumentation-rack', path: '../../instrumentation/rack'
```

Comment:
I'm sure I'm missing something obvious, but why leave the relative import of `instrumentation/rack`?

Reply:
It is non-obvious IMO. The reason is since `action_pack` depends on the `rack` instrumentation it is easier to test changes against the latest version in the repo instead of only relying on a released version.